AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1643,14 +1643,20 @@ - (ASInterfaceState)interfaceState
 
 - (void)setInterfaceState:(ASInterfaceState)interfaceState
 {
-  ASDN::MutexLocker l(_propertyLock);
-  if (interfaceState != _interfaceState) {
-    if ((interfaceState & ASInterfaceStateMeasureLayout) != (_interfaceState & ASInterfaceStateMeasureLayout)) {
+  ASInterfaceState oldValue;
+  {
+    ASDN::MutexLocker l(_propertyLock);
+    oldValue = _interfaceState;
+    _interfaceState = interfaceState;
+  }
+
+  if (interfaceState != oldValue) {
```

Comment:
Small naming suggestion.  How about we name the argument newState, and the local variable oldState?  Probably a bit more clear than oldValue, and it would shorten the comparisons a little.